hive-metastore/src/test/java/org/apache/iceberg/hive/TestHiveCatalog.java

```diff
@@ -272,7 +270,7 @@ public void testCreateTableCustomSortOrder() {
   }
 
   @Test
-  public void testCreateNamespace() throws TException {
+  public void testCreateNamespace() throws Exception {
```

Comment:
I like this change, we can get merged independently of the other parts.

Reply:
+1

If this is a separate fix, then let's move it to a different PR.

Reply:
`hiveLocalDir` in this test is also not cleared. I'll just take care of that and revert irrelevant changes.